source/sync/configure/sync-settings.txt

```diff
@@ -172,12 +202,98 @@ Performance and Storage
+For example, if you have 3 *global queryable fields* and 7 
+*collection queryable fields*, you have 10 queryable fields that apply to the 
+collection.
+
+If you have a field that you only want to query in one collection, but it 
+is configured as a *global queryable field*, this unnecessarily consumes 
+Atlas storage space. For example, if you have a ``user`` field in every 
+collection, but you only use it for Sync queries in one collection, scoping 
+that as a *collection queryable field* reduces storage requirements. Reducing
+the scope means that Sync does not have to maintain metadata for that field
+for the other collections where you are not querying on the ``user`` field.
+
+If you need to reduce storage usage or improve performance, you can remove 
+unneeded queryable fields from your App. However, be aware of the consequences 
+of adding or removing queryable fields. For more information, refer to 
+:ref:`consequences-of-adding-removing-queryable-fields`.
 
-.. seealso::
+For additional considerations, refer to :ref:`optimizing performance and 
+storage when using Flexible Sync <optimizing-performance-and-storage-flexible-sync>`.
 
-   Learn more about :ref:`optimizing performance and storage when using Flexible Sync <optimizing-performance-and-storage-flexible-sync>`.
+.. _fs-indexed-queryable-fields:
+
+Indexed Queryable Fields
+++++++++++++++++++++++++
+
+You can improve performance for certain types of workloads by adding an
+indexed queryable field. An **indexed queryable field** is a 
+**global queryable field** that can be queried on more efficiently, providing
+improved Sync performance. You can designate *one* global queryable field
+as an indexed queryable field.
+
+Indexing a queryable field improves performance for simple queries on a 
+single field, such as ``{“store_id”: 1}`` or  
+``{“user_id”: “641374b03725038381d2e1fb”}``.
+
+The indexed queryable field *must* appear in the schemas of all of your 
+Sync collections, and it must use the same valid data type. For example,
+if your indexed queryable field is ``store_id``, it must appear in all of
+the collections you sync, and it must be the same valid type in all the
+collections. For more information about eligible field types, refer to
+:ref:`flexible-sync-eligible-field-types`.
+
+**You Can't Change Indexed Queryable Field Values on the Client**
+
+After you configure an indexed queryable field, client devices **cannot**
+update an existing object's indexed queryable field value. For example,
+if your indexed queryable field is ``store_id``, the client cannot change 
+this value directly. Changing it from the client is not supported because 
+it may conflict with other updates made to the object in the same timeframe. 
+
+You can still change this value directly in the Atlas database. 
+
+.. warning::
+
+   Avoid changing an object's indexed queryable field value when 
+   possible. It may result in dropping other updates on the same 
```

Comment:
This warning might be a bit scarier than it needs to be. This is not something that we explicitly want user's to avoid doing if it's necessary for their use case, but they should be aware that their write may clobber other updates.

What do you think about a warning like this?

`
Changing an object's indexed queryable field value through Atlas may overwrite concurrent client updates to the object.
`

Reply:
This is a good improvement - thanks! Updating to use this wording.